docs/design/assets/release.uml

```diff
@@ -0,0 +1,72 @@
+@startuml
+'NOTE: use http://sujoyu.github.io/plantuml-previewer to render the Sequence diagram
+
+title Release Workflow
+autonumber
+actor RM order 10
```

Comment:
Any objection against being explicit and calling this actor `Release Manager`?

Reply:
Not from my side, however I'm not sure how it will look in the generated image through